### ISSUE TITLE
Print return values of traced functions

### ISSUE THREAD

**Issue by derekparker** [derekparker/delve#388]
We need to parse and display the return values of traced functions.

**Comment by derekparker** [comment]
Blocked on: https://github.com/golang/go/issues/21100

**Comment by therealplato** [comment]
Would this bug explain this behavior:
```
   250:		actual := w.buf.String()
   251:
=> 252:		if actual != d.expected {
(dlv) p actual
!!!Command failed: could not find symbol value for actual!!!
```

**Comment by aarzilli** [comment]
@therealplato No, you are just using delve with an optimized binary.

**Comment by derekparker** [comment]
Removing the upstream label as this has been fixed for `Go 1.10`. Will need to update DLV to account for this new change.
